mappings/net/minecraft/ChatMessageSender.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_7436 net/minecraft/ChatMessageSender
```

Comment:
Thic class should not be in top-level `net.minecraft` package.

Reply:
Yes, I don't know where to put it :)

Reply:
How about `net.minecraft.network.packet.s2c.play`? We already have `ChunkData` in that package, which is used for `ChunkDataS2CPacket`.

Reply:
imho `ChunkData` is wrong there. In that package should only be packets, not data, etc

Reply:
We can leave it in `net.minecraft.network` for now, as another related class `MessageType` is there as well.